.github/workflows/build_and_test.yml

```diff
@@ -3,10 +3,14 @@ on:
     branches:
       - master
       - r[0-9]+.[0-9]+
+    paths-ignore:
+      - 'experimental/torch_xla2/**'
```

Comment:
Should be `torch_xla/experimental/torch_xla2/**`? Looks like the CI is still triggered.

Reply:
The files are added to `experimental/` and not `torch_xla/experimental` as the latter still builds into the wheel. 

I am guessing the CI triggered because of changing this file (build_and_test.yml). 

We'll find out if this is the case in next PR.